examples/update-viewState/app.js

```diff
@@ -0,0 +1,225 @@
+import React, {useState, useRef, useEffect} from 'react';
```

Comment:
Since this is based on the deckgl buildings example, could you add a comment that links to it here? I think that'll help people who want to learn more.